charts/kyuubi/values.yaml

```diff
@@ -36,6 +36,16 @@ serviceAccount:
   # Specifies ServiceAccount name to be used (created if `create: true`)
   name: ~
 
+# priorityClass used for Kyuubi server pod
+priorityClass:
+  # Specifies whether a priorityClass should be created
+  create: false
+  # Specifies priorityClass name to be used (created if `create: true`)
+  name: ~
+  # half of system-cluster-critical by default
+  value: 1000000000
+  description: "This priority class should be used for kyuubi server pods only."
```

Comment:
I think the content should be the comment instead of the description.

A meaningful description should explain the functionality/requirements of the object itself so that the K8s administers/operators know why the object should exist when they describe the object.

Reply:
helm generate PriorityClass like below

as it used by helm automaticially and has label describe it's managed by Helm, maybe just delete the `description` field and leave comments only?
```
# Source: kyuubi/templates/kyuubi-priorityclass.yaml
apiVersion: scheduling.k8s.io/v1
kind: PriorityClass
metadata:
  name: my-cluster-critical
  labels:
    helm.sh/chart: kyuubi-0.1.0
    app.kubernetes.io/name: kyuubi
    app.kubernetes.io/instance: release-name
    app.kubernetes.io/version: "1.7.1"
    app.kubernetes.io/managed-by: Helm
value: 1e+09
description: This priority class should be used for kyuubi server pods only.

```